### PR TITLE
docs: move about us file and credit contributors

### LINKED ISSUE #11187: Acknowledge maintainers and major contributors in the readme
This is an open source project and I'd like to acknowledge and thank the contributors in the readme. This includes my fellow maintainers (@viceice @JamieMagee) as well as active and repeat contributors such as @HonkingGoose @secustor @ikesyo @zharinov @fgreinacher. We can also have a section to thank those who made significant contributions in the past but have now decreased their involvement for whatever reason so we can avoid any awkward feelings of "dumping" someone from the list who is not contributing enough. Some may even move themselves between sections ;)

Thoughts?

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Move `about-us.md` file from our docs repo to the main Renovate repo
- Credit:
  - Maintainers 
  - Frequent contributors for smaller parts
  - Helpful contributions, even one-offs
- Bump number of contributors to Renovate to `600`

## Context

This PR replaces https://github.com/renovatebot/renovatebot.github.io/pull/173.

Closes #11187.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/about-us.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo
+
+### Valuable contributions
+
+We want to highlight these valuable contributions, even if they are one-offs.
+Some features made a lot of people happy, and efficient!
+
+- [@ikesyo](https://github.com/ikesyo) regularly helpful
+- [@astellingwerf](https://github.com/astellingwerf) reviews PRs
+- [@danports](https://github.com/danports) Worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR.
```
Comment (astellingwerf) [inline docs/usage/about-us.md:37]: All other sentences have a lower case letter after the username, and don't end with a full stop, so:
```suggestion
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
```

**Review by astellingwerf — CHANGES_REQUESTED** [review]
`- [@astellingwerf](https://github.com/astellingwerf) reviews PRs` 😃

File: docs/usage/about-us.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+We want to highlight these valuable contributions, even if they are one-offs.
+Some features made a lot of people happy, and efficient!
+
+- [@ikesyo](https://github.com/ikesyo) regularly helpful
+- [@astellingwerf](https://github.com/astellingwerf) reviews PRs
+- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
+
+## Renovate development
+
+The source code for Renovate is available on GitHub ([renovatebot/renovate](https://github.com/renovatebot/renovate)).
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:41]:
```suggestion
The source code for Renovate is available on [GitHub at `renovatebot/renovate`](https://github.com/renovatebot/renovate).
```

File: docs/usage/about-us.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+This is where we do most of the development.
+
+## About these docs
+
+The Renovate docs are built from Markdown files in [Renovate's Open Source repos](https://github.com/renovatebot/).
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:46]:
```suggestion
The Renovate docs are built from Markdown files in our [`renovatebot/renovate` repository](https://github.com/renovatebot/).
```

File: docs/usage/about-us.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Most of the source files can be found in the [main Renovate repository](https://github.com/renovatebot/renovate/tree/main/docs/usage).
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:47]:
```suggestion
Most of the source files can be found in the [`docs/usage/` directory](https://github.com/renovatebot/renovate/tree/main/docs/usage).
```

**Review by HonkingGoose — COMMENTED** [review]
Let's improve the link names.

File: docs/usage/about-us.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo
+
+### Valuable contributions
+
+We want to highlight these valuable contributions, even if they are one-offs.
+Some features made a lot of people happy, and efficient!
+
+- [@ikesyo](https://github.com/ikesyo) regularly helpful
+- [@astellingwerf](https://github.com/astellingwerf) reviews PRs
+- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
```
Comment (HonkingGoose) [inline docs/usage/about-us.md:37]:
```suggestion
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
- [@shegox](https://github.com/shegox) worked on the Go manager, and improved our docs
```

**Comment by astellingwerf** [comment]
See #15729

**Review by JamieMagee — DISMISSED** [review]
Thank you ❤️ 

Might require changes due to #15729, but approved to prevent blocking

**Review by rarkins — APPROVED** [review]

**Pull request merged**